pennylane/qchem/observable_hf.py

```diff
@@ -138,13 +136,18 @@ def qubit_observable(o_ferm, cutoff=1.0e-12, mapping="jordan_wigner"):
     """
     if mapping == "jordan_wigner":
         h = qml.jordan_wigner(o_ferm, ps=True, tol=cutoff)
+
     elif mapping == "parity":
         qubits = len(o_ferm.wires)
         h = qml.parity_transform(o_ferm, qubits, ps=True, tol=cutoff)
     elif mapping == "bravyi_kitaev":
         qubits = len(o_ferm.wires)
         h = qml.bravyi_kitaev(o_ferm, qubits, ps=True, tol=cutoff)
 
+    if list(h.wires) != sorted(list(h.wires)):
+        h = h + 0 * PauliWord(dict(zip(sorted(h.wires.tolist()), ['X'] * len(h.wires))))
```

Comment:
why do we need this? 🤔

Reply:
We might have terms with wires such as `[0, 1, 2, 4], [0, 1, 3, 5]`, which again lead to wire disordering. Added this term to prevent that and quickly fix the issue for now. Not sure if there is any other simple way to address this without  adding kwargs or altering other functions.